tests/fixtures.py

```diff
@@ -366,6 +368,14 @@ def _make_testerchain():
     eth._utils.headers.GAS_LIMIT_MINIMUM = TEST_GAS_LIMIT
     eth._utils.headers.GENESIS_GAS_LIMIT = TEST_GAS_LIMIT
     eth.vm.forks.frontier.headers.GENESIS_GAS_LIMIT = TEST_GAS_LIMIT
+
+    # Monkey patch to prevent gas estimates
+    def _get_buffered_gas_estimate(web3, transaction, gas_buffer=100000):
+        return TEST_GAS_LIMIT
+
+    import web3
+    web3.eth.get_buffered_gas_estimate = _get_buffered_gas_estimate
```

Comment:
How does this differ from setting a Web3 gas strategy and why not use that API?

Reply:
It's related to gas estimations itself, gas price is independent from that